### PR TITLE
Bump tarantool *.x versions

### PR DESCRIPTION
**Pull request by olegrok**

### CODE REVIEW

File: 2.x/Dockerfile [hunk 1]
```diff
@@ -71,15 +72,6 @@ RUN set -x \
         tcl \
         icu-dev \
         wget \
-    && : "---------- curl ----------" \
```
Comment (Totktonada) [inline 2.x/Dockerfile]: Why only on 2.x?

Comment (olegrok) [reply 2.x/Dockerfile]: Oh, I did not notice that milestone of [this](https://github.com/tarantool/tarantool/issues/4318) issue was changed to 1.10.4

Comment (olegrok) [reply 2.x/Dockerfile]: Amended and pushed again

**Comment by Totktonada** [comment]
Please, verify at least one image locally before merge.

**Pull request merged**